volume/stream_in_out_linux.go

```diff
@@ -26,6 +26,8 @@ func (streamer *tarZstdStreamer) In(tzstInput io.Reader, dest string, privileged
 
 	err = tarCommand.Run()
 	if err != nil {
+		zstdDecompressedStream.Close()
```

Comment:
ignoring err because we're already in an err state and the error from tarcommnad.Run provides more context than the close error. Same applies to all the Closes Ive added in